controlplane/internal/controller/clusterdeployment_controller.go

```diff
@@ -327,5 +328,14 @@ func (r *ClusterDeploymentReconciler) computeAgentClusterInstall(
 			ManifestsConfigMapRefs: additionalManifests,
 		},
 	}
+
+	if len(acp.Spec.AgentConfigSpec.APIVIPs) > 0 && len(acp.Spec.AgentConfigSpec.IngressVIPs) > 0 {
```

Comment:
Also should we have a check that if it's a multinode cluster, there should be VIPs set?

Reply:
I don't think so, it could be UMN, right?